src/libostree/ostree-repo-static-delta-compilation.c

```diff
@@ -1446,10 +1457,12 @@ ostree_repo_static_delta_generate (OstreeRepo                   *self,
     goto out;
 
   descriptor_builder = ot_variant_builder_new (G_VARIANT_TYPE (OSTREE_STATIC_DELTA_SUPERBLOCK_FORMAT), descriptor_tmpf.fd);
+  desc_sign_builder = g_variant_builder_new (G_VARIANT_TYPE (OSTREE_STATIC_DELTA_SUPERBLOCK_FORMAT));
```

Comment:
Why we need to construct `desc_sign_builder` and `desc_sign_v` if there is no need to sign data? I.e. if `opt_sign_name == NULL ||  opt_key_ids == NULL`